config-test/webextensions-check-prod.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -x
+
+export FXA_USER_EMAIL="chartjes@mozilla.com"
+
+export SERVER_URL="https://webextensions.settings.services.mozilla.com/v1"
```

Comment:
where possible, i'd still recommend moving your constants off to a manifest.ini file. things like your user email or server DNS we're putting into manifest files with clearly marked sections for [dev]  [stage] [prod], that way the tests can easily be invoked with different test envs and parameters.